scripts/puppeteer.server.js

```diff
@@ -0,0 +1,9 @@
+import * as http from 'http';
```

Comment:
If you can just import `createServer` since that's all we need
```suggestion
import {createServer} from 'http';
```